DDD-8.md

```diff
@@ -126,3 +126,146 @@ So if the snapshot is reading a chunk from a table that is modified through a su
 Furthermore, the de-duplication algorithm is based on the assumption that the transaction isolation level is set to [READ COMMITTED](https://www.postgresql.org/docs/current/transaction-iso.html).
 
 Since the support for `pg_current_snapshot()` was added on PostgreSQL 13, the read-only incremental snapshot will be available only for PostgreSQl versions greater than or equal to 13. 
+
+### MongoDB
+
+Also for MongoDB the motivation behind the research a possible way to provide read-only incremental snapshot are the same
+with a specific addition: try to overcome the challenges related to the different kind of MongoDB topologies. 
+
+#### Topologies
+MongoDB has three topology types:
+
+* Standalone server
+* Replica set
+* Sharded cluster
+
+Since Standalone server do not have the _oplog_ and so Debezium is not able to streaming events from it, we do not consider it. Although MongoDB does not recommend running a standalone server in production.
+
+##### Replica set
+
+A [replica set (RS)](https://www.mongodb.com/docs/manual/core/replica-set-members/) in MongoDB is a group of _mongod_ processes that provide redundancy and high availability. The members of a replica set are:
+
+* Primary:
+  receives all write operations.
+* Secondaries:
+  replicate operations from the primary to maintain an identical data set. To replicate data, a secondary applies operations from the primary's oplog to its own data set in an asynchronous process. A replica set can have one or more secondaries.
```

Comment:
Technically there are also hidden members (secondaries which are purposefully hidden from client advertising) and arbiter nodes (nodes which don't hold data but vote in primary election). Neither of these are likely relevant for our case (although technically a hidden node could be used for CDC with direct connection)